world-chain-builder/crates/world/payload/src/stamp.rs

```diff
@@ -1,58 +1,60 @@
+use std::collections::HashSet;
+
+use crate::builder::WorldChainPayloadBuilderCtx;
 use alloy::sol;
-use alloy_network::{EthereumWallet, NetworkWallet, TransactionBuilder};
-use alloy_signer_local::PrivateKeySigner;
+use alloy_consensus::SignableTransaction;
+use alloy_network::{TransactionBuilder, TxSignerSync};
 use eyre::eyre::eyre;
-use futures::executor::block_on;
-use op_alloy_consensus::OpTxEnvelope;
-use op_alloy_network::Optimism;
 use op_alloy_rpc_types::OpTransactionRequest;
+use op_revm::OpContext;
 use reth_evm::Evm;
 use reth_optimism_node::OpEvm;
-use WorldChainBlockRegistry::stampBlockCall;
-
-use crate::{builder::WorldChainPayloadBuilderCtx, inspector::PBHCallTracer};
+use reth_optimism_primitives::OpTransactionSigned;
+use reth_primitives::Recovered;
+use reth_primitives_traits::SignedTransaction;
+use revm::Inspector;
+use semaphore_rs::Field;
+use WorldChainBlockRegistry::spendNullifierHashesCall;
 
 sol! {
     #[sol(rpc)]
     interface WorldChainBlockRegistry {
```

Comment:
Can we update this interface name since the block registry will not be used anymore?